pageserver/src/lib.rs

```diff
@@ -254,3 +267,36 @@ mod backoff_defaults_tests {
         );
     }
 }
+
+#[cfg(test)]
+mod timed_tests {
+    use super::timed;
+    use std::time::Duration;
+
+    #[tokio::test]
+    async fn test_timed() {
```

Comment:
```suggestion
    async fn timed_completes_when_given_future_completes() {
```

The `test_` prefixing is a python thing, in rust we find tests by the `#[test]` which is applied by `#[tokio::test]` proc_macro wrapper.